docs/scripts/gen_mds.py

```diff
@@ -33,10 +41,9 @@
     env_name = split[1]
     filtered_envs_by_type[env_name] = env_spec
 
-
 filtered_envs = {
-    k.split(":")[1]: v
-    for k, v in sorted(
+    env[0]: env[1]
+    for env in sorted(
```

Comment:
env: env_spec
for env, env_spec in sorted(...

Reply:
see my hosted docs: https://fenggu.me/MiniGrid/
For the first issue, Gym v0.26 is already replaced by Gymnasium v0.26, because this index page directly comes from README
For the second issue, I originally have for env, env_spec in sorted(..., which causes the CI issue. Changing it to for env in sorted(... solves the problem

Reply:
Why does the docs/index.md exist then if it isn't used? 
Ok, don't worry on the code change proposal

Reply:
It was originally there because I haven't automate the index.md generation. I'll remove it and some other unused files in the next PR.